src/pois.rs

```diff
@@ -176,6 +176,18 @@ impl IndexedPoi {
             &country_codes,
             langs,
         );
+        let mut self_country_labels_to_add = Vec::new();
+        for country_code in country_codes.iter() {
+            if !self.poi.labels.0.iter().any(|x| x.key == *country_code) {
+                self_country_labels_to_add.push(country_code.to_lowercase());
+            }
+        }
+        for country_code in self_country_labels_to_add {
+            self.poi.labels.0.push(Property {
+                key: country_code,
+                value: self.poi.name.clone(),
+            });
```

Comment:
It doesn't make sense for me to use a `country_code` as `key`  here.  
`poi.labels` are supposed to contain labels for each *language* code. That way, the geocoder can filter or boost the results according to the `lang` parameter used in the request.

Am I missing something ?